src/algo/depth-to-rgb-calibration/calibration.h

```diff
@@ -144,6 +144,7 @@ namespace depth_to_rgb_calibration {
         calib( calib const & ) = default;
         explicit calib( rs2_intrinsics_double const & rgb_intrinsics, rs2_extrinsics_double const & depth_to_rgb_extrinsics);
         explicit calib( rs2_intrinsics const & rgb_intrinsics, rs2_extrinsics const & depth_to_rgb_extrinsics);
+        explicit calib( rs2_intrinsics_double const & rgb_intrinsics, rs2_extrinsics const & depth_to_rgb_extrinsics );
```

Comment:
Why is this needed? We have an implicit rs2_extrinsics_double ctor, it's not good?

Reply:
Removed